src/features/student/views/StudentToolsTracesView/components/StudentTraceAssociations/StudentTraceAssociations.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import type { AdditionalSkillAssociationDTO, SkillLevelAssociationDTO } from '@/api/avenir-esr'
+import { StudentTraceAdditionalSkillAssociationCard, StudentTraceSkillLevelAssociationCard } from '@/features/student/components/cards'
+import { AvIconText, MDI_ICONS } from '@avenirs-esr/avenirs-dsav'
+
+export interface StudentTraceAssociationsProps {
+  skillLevelAssociations: SkillLevelAssociationDTO[]
+  additionalSkillAssociations: AdditionalSkillAssociationDTO[]
+}
+
+const { skillLevelAssociations, additionalSkillAssociations } = defineProps<StudentTraceAssociationsProps>()
+
+const totalAssociations = computed(() => skillLevelAssociations.length + additionalSkillAssociations.length)
+
+const skillColors = [
+  'var(--skill1)',
+  'var(--skill2)',
+  'var(--skill3)',
+  'var(--skill4)',
+  'var(--skill5)',
+  'var(--skill6)',
+  'var(--skill7)',
+  'var(--skill8)',
+  'var(--skill9)',
+  'var(--skill10)',
+  'var(--skill11)',
+  'var(--skill12)'
+]
+
+function getSkillColor (index: number): string {
+  return skillColors[index % skillColors.length]
+}
```

Comment:
```suggestion
function getSkillColor (index: number): string {
  return `var(--skill${index % 12})`
}
```
Je pense que tu pouvais faire ça directement